src/l500/l500-options.cpp

```diff
@@ -164,6 +165,11 @@ namespace librealsense
         case RS2_L500_VISUAL_PRESET_CUSTOM:
             move_to_custom();
             break;
+        case RS2_L500_VISUAL_PRESET_DEFAULT:
+            throw  invalid_value_exception(to_string() << "the Default preset signifies that the controls have not been changed \n"
+                                                           "since initialization, the API does not support changing back to this state.\n"
+                                                           "Please choose one of the other presets");
+            break;
         default: break;
```

Comment:
Add to log as it seem to be an error case

Reply:
Done.